docs/modules/release-notes/pages/all-releases/4_14.adoc

```diff
@@ -0,0 +1,117 @@
+= Kobiton 4.14 release notes
+:navtitle: Kobiton 4.14 release notes
+
+_August 2, 2024_
+
+== New OS support
+
+Kobiton 4.14 adds support for the following:
+
+* iOS/iPadOS 18 Beta 3
+* Xcode 16 Beta
```

Comment:
Please add supported versions to supported-platforms/mobile-operating-systems-and-devices

Reply:
Added.